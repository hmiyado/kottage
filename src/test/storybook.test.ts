import initStoryshots, {
  Stories2SnapsConverter,
} from '@storybook/addon-storyshots'
import { setupServer } from 'msw/node'
import React from 'react'
import { act, create } from 'react-test-renderer'

//https://github.com/storybookjs/storybook/issues/8083#issuecomment-533184737
jest.mock('global', () =>
  Object.assign(global, { window: { STORYBOOK_HOOKS_CONTEXT: '' } })
)

// https://github.com/storybookjs/storybook/issues/7745#issuecomment-807338022
const converter = new Stories2SnapsConverter()
initStoryshots({
  test: async ({ story, done }) => {
    try {
      let renderer
      await act(() => {
        // React.createElement() is important because of hooks [shouldn't call story.render() directly]
        renderer = create(React.createElement(story.render))
      })

      // wait for state changes
      await act(() => new Promise((resolve) => setTimeout(resolve, 0)))

      expect(renderer).toMatchSnapshot()

      done?.()
    } finally {
    }
  },
})
