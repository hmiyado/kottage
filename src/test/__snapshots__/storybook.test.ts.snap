// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Contact Primary 1`] = `
<section
  className="section"
>
  <h2
    className="headline"
  >
    Skill
  </h2>
  <section
    className="body"
  >
    Android/Kotlin エンジニア。その他 Swift/iOS JavaScript, TypeScript, Ruby など。詳細は
     
    <a
      className="link"
      href="https://github.com/hmiyado/resume"
    >
      GitHub レジュメ
    </a>
     
    を参照。
  </section>
  <h2
    className="headline"
  >
    Price
  </h2>
  <section
    className="body"
  >
    時間単価6000円〜。詳細要相談。
  </section>
  <h2
    className="headline"
  >
    Contact
  </h2>
  <section
    className="body"
  >
    <a
      className="link"
      href="https://twitter.com/miyado20th"
    >
      Twitter
    </a>
  </section>
</section>
`;

exports[`Storyshots Entry Entry 1 1`] = `
<article>
  <h2
    className="headline"
  >
    <a
      className="titleLink"
      href="/entries/1"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      GW
    </a>
  </h2>
  <section
    className="body"
  >
    GW はコーディングしたり、リモートゲームしたり、無計画だったわりには楽しく過ごせた。コーディングでは2年前のリポジトリを引っ張り出して整理したら、思っていた以上に興が乗った。次に触るのは2年後のGWかもしれない。
  </section>
  <div
    className="footer"
  >
    <div
      className="body1 onSurface onSurfaceMedium"
    >
      hmiyado 2021-05-05T21:12
    </div>
    <a
      className="link"
      href="/entries/1"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <div
        className="body1 onSurface onSurfaceMedium"
      >
        コメント 10
      </div>
    </a>
  </div>
</article>
`;

exports[`Storyshots Entry Primary 1`] = `
<article>
  <h2
    className="headline"
  >
    <a
      className="titleLink"
      href="/entries/1"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      short title
    </a>
  </h2>
  <section
    className="body"
  >
    body
  </section>
  <div
    className="footer"
  >
    <div
      className="body1 onSurface onSurfaceMedium"
    >
      hmiyado 2021-05-05T21:12
    </div>
    <a
      className="link"
      href="/entries/1"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <div
        className="body1 onSurface onSurfaceMedium"
      >
        コメント 0
      </div>
    </a>
  </div>
</article>
`;

exports[`Storyshots EntryForm Primary 1`] = `
<div
  className="container"
>
  <fieldset
    className="container"
  >
    <input
      defaultChecked={true}
      id="edit"
      name="editMode"
      onChange={[Function]}
      type="radio"
    />
    <label
      htmlFor="edit"
    >
      編集
    </label>
    <input
      defaultChecked={false}
      id="preview"
      name="editMode"
      onChange={[Function]}
      type="radio"
    />
    <label
      htmlFor="preview"
    >
      プレビュー
    </label>
  </fieldset>
  <div>
    <div
      className="formContainer"
    >
      <label
        className="label"
        htmlFor="タイトル"
      >
        タイトル
      </label>
      <div
        className="textLine"
      >
        <input
          className="input"
          id="タイトル"
          name="タイトル"
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="assistiveText"
    />
  </div>
  <div
    className="formContainer"
  >
    <label
      className="label"
      htmlFor="本文"
    >
      本文
    </label>
    <textarea
      className="textarea"
      id="本文"
      name="本文"
      onChange={[Function]}
      value=""
    />
  </div>
  <div
    className="footer"
  >
    <button
      className="button"
      disabled={true}
      onClick={[Function]}
    >
      投稿する
    </button>
    <button
      className="button"
    >
      キャンセル
    </button>
  </div>
</div>
`;

exports[`Storyshots Error Primary 1`] = `
<section
  className="section"
>
  <h2
    className="headline"
  >
    404 - Not Found
  </h2>
  <section
    className="body"
  >
    お探しのページは見つかりませんでした
  </section>
</section>
`;

exports[`Storyshots atoms/Avatar Override Class 1`] = `
<img
  alt="avatar"
  className="icon icon"
  src="/components/pieces/avatar/miyado_icon.svg"
/>
`;

exports[`Storyshots atoms/Avatar Primary 1`] = `
<img
  alt="avatar"
  className="icon "
  src="/components/pieces/avatar/miyado_icon.svg"
/>
`;

exports[`Storyshots atoms/Button Disabled 1`] = `
<button
  className="button"
  disabled={true}
>
  BUTTON
</button>
`;

exports[`Storyshots atoms/Button No Icon 1`] = `
<button
  className="button"
>
  BUTTON
</button>
`;

exports[`Storyshots atoms/Sentence Long 1`] = `
<div
  style={
    Object {
      "width": "480px",
    }
  }
>
  <h2
    className="headline"
  >
    long long long long long long title
  </h2>
  <section
    className="body"
  />
</div>
`;

exports[`Storyshots atoms/Sentence Long Word Body 1`] = `
<div
  style={
    Object {
      "width": "480px",
    }
  }
>
  <h2
    className="headline"
  >
    long word body
  </h2>
  <section
    className="body"
  >
    line1
superlonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglongword

  </section>
</div>
`;

exports[`Storyshots atoms/Sentence Markdown 1`] = `
<div
  style={
    Object {
      "width": "480px",
    }
  }
>
  <h2
    className="headline"
  >
    long word body
  </h2>
  <section
    className="body"
  >
    
  # Heading 1
  
  ## Heading 2

  ### Heading 3

  #### Heading 4

  ##### Heading 5

  ###### Heading 6

  ## unorderd list
  
  * Item 1
      * Item 1-1
          * Item 1-1-1
  * Item 2
  * Item 3
  
  ## ordered list
  
  1. first
  1. second 
  1. third

  ## styles

  *Italic*

  **Bold**

  [Link(github)](https://github.com)

  ![Image](./public/components/pieces/avatar/miyado_icon.svg)

  &gt;Block quote

  &gt;&gt;Indented Block quote

  Horizontal rule

  ---

  \`Inline code\`

  ## code highlight
  
  ### JavaScript

  \`\`\`js
  import React from 'react'
  import ReactDOM from 'react-dom'
  import ReactMarkdown from 'react-markdown'
  import rehypeHighlight from 'rehype-highlight'

  ReactDOM.render(
    &lt;ReactMarkdown rehypePlugins={[rehypeHighlight]}&gt;{'# Your markdown here'}&lt;/ReactMarkdown&gt;,
    document.querySelector('#content')
  )
  \`\`\`

  ### Kotlin

  \`\`\`kotlin
  data class ClientSession(
      val token: String
  ): CsrfTokenBoundClient {
      override val representation: String = token
  }
  \`\`\`

  
  </section>
</div>
`;

exports[`Storyshots atoms/Sentence Multi Lines 1`] = `
<div
  style={
    Object {
      "width": "480px",
    }
  }
>
  <h2
    className="headline"
  >
    long title
  </h2>
  <section
    className="body"
  >
    line1
line2line2line2
line3 and two line breaks

line4
  </section>
</div>
`;

exports[`Storyshots atoms/Sentence Short 1`] = `
<div
  style={
    Object {
      "width": "480px",
    }
  }
>
  <h2
    className="headline"
  >
    short title
  </h2>
  <section
    className="body"
  />
</div>
`;

exports[`Storyshots atoms/TextArea Primary 1`] = `
<div
  className="formContainer"
>
  <label
    className="label"
    htmlFor="Label"
  >
    Label
  </label>
  <textarea
    className="textarea"
    id="Label"
    name="Label"
  />
</div>
`;

exports[`Storyshots atoms/TextField No Icon 1`] = `
<div>
  <div
    className="formContainer"
  >
    <label
      className="label"
      htmlFor="Label"
    >
      Label
    </label>
    <div
      className="textLine"
    >
      <input
        className="input"
        id="Label"
        name="Label"
        type="text"
      />
    </div>
  </div>
  <div
    className="assistiveText"
  >
    Assistive text
  </div>
</div>
`;

exports[`Storyshots atoms/TextField No Icon No Assistive Text 1`] = `
<div>
  <div
    className="formContainer"
  >
    <label
      className="label"
      htmlFor="Label"
    >
      Label
    </label>
    <div
      className="textLine"
    >
      <input
        className="input"
        id="Label"
        name="Label"
        type="text"
      />
    </div>
  </div>
  <div
    className="assistiveText"
  />
</div>
`;

exports[`Storyshots comment/Comment Primary 1`] = `
<article>
  <h2
    className="headline"
  />
  <section
    className="body"
  >
    さっき買い物から帰ってきたところなのにまた買い出しにいく敗北感と面倒臭さに勝てませんでして．．．
  </section>
  <div
    className="footer"
  >
    <div
      className="caption onSurface onSurfaceMedium"
    >
      みやど 2021-12-30T23:12:51+0900
    </div>
  </div>
</article>
`;

exports[`Storyshots comment/CommentForm Primary 1`] = `
<div
  className="container"
>
  <div>
    <div
      className="formContainer"
    >
      <label
        className="label"
        htmlFor="Name"
      >
        Name
      </label>
      <div
        className="textLine"
      >
        <input
          className="input"
          id="Name"
          name="Name"
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="assistiveText"
    />
  </div>
  <div
    className="formContainer"
  >
    <label
      className="label"
      htmlFor="Body"
    >
      Body
    </label>
    <textarea
      className="textarea"
      id="Body"
      name="Body"
      onChange={[Function]}
      value=""
    />
  </div>
  <div
    className="footer"
  >
    <button
      className="button"
      disabled={true}
      onClick={[Function]}
    >
      SUBMIT
    </button>
  </div>
</div>
`;

exports[`Storyshots comment/CommentList No Comments 1`] = `
<div
  className="container"
>
  <div
    className="formContainer"
  >
    <div
      className="container"
    >
      <div>
        <div
          className="formContainer"
        >
          <label
            className="label"
            htmlFor="Name"
          >
            Name
          </label>
          <div
            className="textLine"
          >
            <input
              className="input"
              id="Name"
              name="Name"
              onChange={[Function]}
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          className="assistiveText"
        />
      </div>
      <div
        className="formContainer"
      >
        <label
          className="label"
          htmlFor="Body"
        >
          Body
        </label>
        <textarea
          className="textarea"
          id="Body"
          name="Body"
          onChange={[Function]}
          value=""
        />
      </div>
      <div
        className="footer"
      >
        <button
          className="button"
          disabled={true}
          onClick={[Function]}
        >
          SUBMIT
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots comment/CommentList Seven Comments 1`] = `
<div
  className="container"
>
  <article>
    <h2
      className="headline"
    />
    <section
      className="body"
    >
      aaa
    </section>
    <div
      className="footer"
    >
      <div
        className="caption onSurface onSurfaceMedium"
      >
        aaa 2021-12-30T18:12:37+0900
      </div>
    </div>
  </article>
  <article>
    <h2
      className="headline"
    />
    <section
      className="body"
    >
      b
    </section>
    <div
      className="footer"
    >
      <div
        className="caption onSurface onSurfaceMedium"
      >
        a 2021-12-30T18:27:22+0900
      </div>
    </div>
  </article>
  <article>
    <h2
      className="headline"
    />
    <section
      className="body"
    >
      bbbbb
    </section>
    <div
      className="footer"
    >
      <div
        className="caption onSurface onSurfaceMedium"
      >
        aaa 2021-12-30T18:29:58+0900
      </div>
    </div>
  </article>
  <article>
    <h2
      className="headline"
    />
    <section
      className="body"
    >
      b
    </section>
    <div
      className="footer"
    >
      <div
        className="caption onSurface onSurfaceMedium"
      >
        a 2021-12-30T18:31:42+0900
      </div>
    </div>
  </article>
  <article>
    <h2
      className="headline"
    />
    <section
      className="body"
    >
      bbbbb
    </section>
    <div
      className="footer"
    >
      <div
        className="caption onSurface onSurfaceMedium"
      >
        aaaa 2021-12-30T20:28:46+0900
      </div>
    </div>
  </article>
  <article>
    <h2
      className="headline"
    />
    <section
      className="body"
    >
      ああああ
    </section>
    <div
      className="footer"
    >
      <div
        className="caption onSurface onSurfaceMedium"
      >
        hogehgoe 2021-12-30T22:08:00+0900
      </div>
    </div>
  </article>
  <article>
    <h2
      className="headline"
    />
    <section
      className="body"
    >
      するぞー
    </section>
    <div
      className="footer"
    >
      <div
        className="caption onSurface onSurfaceMedium"
      >
        コメント 2021-12-30T22:09:12+0900
      </div>
    </div>
  </article>
  <div
    className="formContainer"
  >
    <div
      className="container"
    >
      <div>
        <div
          className="formContainer"
        >
          <label
            className="label"
            htmlFor="Name"
          >
            Name
          </label>
          <div
            className="textLine"
          >
            <input
              className="input"
              id="Name"
              name="Name"
              onChange={[Function]}
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          className="assistiveText"
        />
      </div>
      <div
        className="formContainer"
      >
        <label
          className="label"
          htmlFor="Body"
        >
          Body
        </label>
        <textarea
          className="textarea"
          id="Body"
          name="Body"
          onChange={[Function]}
          value=""
        />
      </div>
      <div
        className="footer"
      >
        <button
          className="button"
          disabled={true}
          onClick={[Function]}
        >
          SUBMIT
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots comment/CommentLoading Primary 1`] = `
<article
  className="container"
>
  <section
    className="body"
  />
  <div
    className="footer"
  />
</article>
`;

exports[`Storyshots layout/Footer Primary 1`] = `
<footer
  className="container"
>
  <div
    className="text"
  >
    (C) 2022 miyado
  </div>
  <a
    className="link"
    href="/contact"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    contact
  </a>
</footer>
`;

exports[`Storyshots layout/Header Primary 1`] = `
<header
  className="header"
>
  <h1
    className="title"
  >
    <a
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      miyado.dev
    </a>
  </h1>
</header>
`;

exports[`Storyshots page/pagenavigation/PageNavigation First 1`] = `
<nav
  className="container"
>
  <span
    className="disabledLink"
  >
    |&lt; 最新
  </span>
  <span
    className="disabledLink"
  >
    &lt; 前へ
  </span>
  <span
    className="currentPage"
  >
    1
  </span>
  <a
    className="link"
    href="/pages/2"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    次へ &gt;
  </a>
  <a
    className="link"
    href="/pages/10"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    最終 &gt;|
  </a>
</nav>
`;

exports[`Storyshots page/pagenavigation/PageNavigation Last Page 1`] = `
<nav
  className="container"
>
  <a
    className="link"
    href="/pages/1"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    |&lt; 最新
  </a>
  <a
    className="link"
    href="/pages/9"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    &lt; 前へ
  </a>
  <span
    className="currentPage"
  >
    10
  </span>
  <span
    className="disabledLink"
  >
    次へ &gt;
  </span>
  <span
    className="disabledLink"
  >
    最終 &gt;|
  </span>
</nav>
`;

exports[`Storyshots page/pagenavigation/PageNavigation Primary 1`] = `
<nav
  className="container"
>
  <a
    className="link"
    href="/pages/1"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    |&lt; 最新
  </a>
  <a
    className="link"
    href="/pages/4"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    &lt; 前へ
  </a>
  <span
    className="currentPage"
  >
    5
  </span>
  <a
    className="link"
    href="/pages/6"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    次へ &gt;
  </a>
  <a
    className="link"
    href="/pages/10"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    最終 &gt;|
  </a>
</nav>
`;

exports[`Storyshots pieces/SegmentedButton Item 2 1`] = `
<fieldset
  className="container"
>
  <input
    defaultChecked={false}
    id="first"
    name="item2"
    onChange={[Function]}
    type="radio"
  />
  <label
    htmlFor="first"
  >
    一番目
  </label>
  <input
    defaultChecked={false}
    id="second"
    name="item2"
    onChange={[Function]}
    type="radio"
  />
  <label
    htmlFor="second"
  >
    二番目
  </label>
</fieldset>
`;

exports[`Storyshots pieces/SegmentedButton Item 7 1`] = `
<fieldset
  className="container"
>
  <input
    defaultChecked={false}
    id="sunday"
    name="item7"
    onChange={[Function]}
    type="radio"
  />
  <label
    htmlFor="sunday"
  >
    Sunday
  </label>
  <input
    defaultChecked={false}
    id="monday"
    name="item7"
    onChange={[Function]}
    type="radio"
  />
  <label
    htmlFor="monday"
  >
    Monday
  </label>
  <input
    defaultChecked={false}
    id="tuesday"
    name="item7"
    onChange={[Function]}
    type="radio"
  />
  <label
    htmlFor="tuesday"
  >
    Tuesday
  </label>
  <input
    defaultChecked={false}
    id="wednesday"
    name="item7"
    onChange={[Function]}
    type="radio"
  />
  <label
    htmlFor="wednesday"
  >
    Wednesday
  </label>
  <input
    defaultChecked={false}
    id="thursday"
    name="item7"
    onChange={[Function]}
    type="radio"
  />
  <label
    htmlFor="thursday"
  >
    Thursday
  </label>
  <input
    defaultChecked={false}
    id="friday"
    name="item7"
    onChange={[Function]}
    type="radio"
  />
  <label
    htmlFor="friday"
  >
    Friday
  </label>
  <input
    defaultChecked={false}
    id="saturday"
    name="item7"
    onChange={[Function]}
    type="radio"
  />
  <label
    htmlFor="saturday"
  >
    Saturday
  </label>
</fieldset>
`;

exports[`Storyshots sidemenu/Profile Primary 1`] = `
<div
  className="container"
>
  <img
    alt="avatar"
    className="icon icon"
    src="/components/pieces/avatar/miyado_icon.svg"
  />
  <div
    className="textContainer"
  >
    <div
      className="handlename"
    >
      miyado20th
    </div>
    <div
      className="description"
    >
      アノマロカリス
    </div>
  </div>
</div>
`;

exports[`Storyshots sidemenu/ServiceReference Dark 1`] = `
<div
  className="undefined icons"
>
  <a
    href="https://github.com/hmiyado"
    rel="noreferrer"
    target="_blank"
  >
    <img
      alt="GitHub"
      height={32}
      src={
        Object {
          "blurDataURL": "data:image/png;base64,imagedata",
          "height": 24,
          "src": "/img.jpg",
          "width": 24,
        }
      }
      width={32}
    />
  </a>
  <a
    href="https://qiita.com/hmiyado"
    rel="noreferrer"
    target="_blank"
  >
    <img
      alt="Qiita"
      height={32}
      src={
        Object {
          "blurDataURL": "data:image/png;base64,imagedata",
          "height": 24,
          "src": "/img.jpg",
          "width": 24,
        }
      }
      width={32}
    />
  </a>
  <a
    href="https://twitter.com/miyado20th"
    rel="noreferrer"
    target="_blank"
  >
    <img
      alt="Twitter"
      height={32}
      src={
        Object {
          "blurDataURL": "data:image/png;base64,imagedata",
          "height": 24,
          "src": "/img.jpg",
          "width": 24,
        }
      }
      width={32}
    />
  </a>
  <a
    href="/feed/atom.xml"
    rel="noreferrer"
    target="_blank"
  >
    <img
      alt="Feed"
      height={32}
      src={
        Object {
          "blurDataURL": "data:image/png;base64,imagedata",
          "height": 24,
          "src": "/img.jpg",
          "width": 24,
        }
      }
      width={32}
    />
  </a>
</div>
`;

exports[`Storyshots sidemenu/ServiceReference Light 1`] = `
<div
  className="undefined icons"
>
  <a
    href="https://github.com/hmiyado"
    rel="noreferrer"
    target="_blank"
  >
    <img
      alt="GitHub"
      height={32}
      src={
        Object {
          "blurDataURL": "data:image/png;base64,imagedata",
          "height": 24,
          "src": "/img.jpg",
          "width": 24,
        }
      }
      width={32}
    />
  </a>
  <a
    href="https://qiita.com/hmiyado"
    rel="noreferrer"
    target="_blank"
  >
    <img
      alt="Qiita"
      height={32}
      src={
        Object {
          "blurDataURL": "data:image/png;base64,imagedata",
          "height": 24,
          "src": "/img.jpg",
          "width": 24,
        }
      }
      width={32}
    />
  </a>
  <a
    href="https://twitter.com/miyado20th"
    rel="noreferrer"
    target="_blank"
  >
    <img
      alt="Twitter"
      height={32}
      src={
        Object {
          "blurDataURL": "data:image/png;base64,imagedata",
          "height": 24,
          "src": "/img.jpg",
          "width": 24,
        }
      }
      width={32}
    />
  </a>
  <a
    href="/feed/atom.xml"
    rel="noreferrer"
    target="_blank"
  >
    <img
      alt="Feed"
      height={32}
      src={
        Object {
          "blurDataURL": "data:image/png;base64,imagedata",
          "height": 24,
          "src": "/img.jpg",
          "width": 24,
        }
      }
      width={32}
    />
  </a>
</div>
`;

exports[`Storyshots sidemenu/SideMenu Primary 1`] = `
<aside
  className="container undefined"
>
  <div
    className="profile"
    onClick={[Function]}
  >
    <div
      className="container"
    >
      <img
        alt="avatar"
        className="icon icon"
        src="/components/pieces/avatar/miyado_icon.svg"
      />
      <div
        className="textContainer"
      >
        <div
          className="handlename"
        >
          miyado20th
        </div>
        <div
          className="description"
        >
          アノマロカリス
        </div>
      </div>
    </div>
  </div>
  <div
    className="servicereference icons"
  >
    <a
      href="https://github.com/hmiyado"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="GitHub"
        height={32}
        src={
          Object {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 24,
            "src": "/img.jpg",
            "width": 24,
          }
        }
        width={32}
      />
    </a>
    <a
      href="https://qiita.com/hmiyado"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="Qiita"
        height={32}
        src={
          Object {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 24,
            "src": "/img.jpg",
            "width": 24,
          }
        }
        width={32}
      />
    </a>
    <a
      href="https://twitter.com/miyado20th"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="Twitter"
        height={32}
        src={
          Object {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 24,
            "src": "/img.jpg",
            "width": 24,
          }
        }
        width={32}
      />
    </a>
    <a
      href="/feed/atom.xml"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="Feed"
        height={32}
        src={
          Object {
            "blurDataURL": "data:image/png;base64,imagedata",
            "height": 24,
            "src": "/img.jpg",
            "width": 24,
          }
        }
        width={32}
      />
    </a>
  </div>
  <div
    className="userform"
  />
</aside>
`;

exports[`Storyshots sidemenu/UserForm Sign In 1`] = `
<div
  className="container"
>
  <div>
    <div
      className="formContainer"
    >
      <label
        className="label"
        htmlFor="ID"
      >
        ID
      </label>
      <div
        className="textLine"
      >
        <input
          className="input"
          id="ID"
          name="ID"
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="assistiveText"
    />
  </div>
  <div>
    <div
      className="formContainer"
    >
      <label
        className="label"
        htmlFor="Password"
      >
        Password
      </label>
      <div
        className="textLine"
      >
        <input
          className="input"
          id="Password"
          name="Password"
          onChange={[Function]}
          type="password"
          value=""
        />
      </div>
    </div>
    <div
      className="assistiveText"
    />
  </div>
  <div
    className="buttonContainer"
  >
    <button
      className="button"
      disabled={true}
      onClick={[Function]}
    >
      SIGN IN
    </button>
    <button
      className="button"
      disabled={true}
      onClick={[Function]}
    >
      SIGN UP
    </button>
  </div>
</div>
`;

exports[`Storyshots sidemenu/UserForm Sign Out 1`] = `
<div
  className="container"
>
  <div
    className="account"
  >
    <div
      className="label"
    >
      Name
    </div>
    <div
      className="body"
    >
      users-current
    </div>
  </div>
  <button
    className="button"
    onClick={[Function]}
  >
    SIGN OUT
  </button>
</div>
`;

exports[`Storyshots sidemenu/userform/SignInForm Primary 1`] = `
<div
  className="container"
>
  <div>
    <div
      className="formContainer"
    >
      <label
        className="label"
        htmlFor="ID"
      >
        ID
      </label>
      <div
        className="textLine"
      >
        <input
          className="input"
          id="ID"
          name="ID"
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="assistiveText"
    />
  </div>
  <div>
    <div
      className="formContainer"
    >
      <label
        className="label"
        htmlFor="Password"
      >
        Password
      </label>
      <div
        className="textLine"
      >
        <input
          className="input"
          id="Password"
          name="Password"
          onChange={[Function]}
          type="password"
          value=""
        />
      </div>
    </div>
    <div
      className="assistiveText"
    />
  </div>
  <div
    className="buttonContainer"
  >
    <button
      className="button"
      disabled={true}
      onClick={[Function]}
    >
      SIGN IN
    </button>
    <button
      className="button"
      disabled={true}
      onClick={[Function]}
    >
      SIGN UP
    </button>
  </div>
</div>
`;

exports[`Storyshots sidemenu/userform/SignOutForm Primary 1`] = `
<div
  className="container"
>
  <div
    className="account"
  >
    <div
      className="label"
    >
      Name
    </div>
    <div
      className="body"
    >
      ScreenName
    </div>
  </div>
  <button
    className="button"
  >
    SIGN OUT
  </button>
</div>
`;

exports[`Storyshots sidemenu/userform/userformloading Primary 1`] = `
<div
  className="container"
>
  <div
    className="content"
  />
  <div
    className="content"
  />
</div>
`;

exports[`Storyshots template/Layout Primary 1`] = `
<div
  className="container"
>
  <header
    className="header"
  >
    <h1
      className="title"
    >
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        miyado.dev
      </a>
    </h1>
  </header>
  <div
    className="main"
  >
    <div>
      test
    </div>
  </div>
  <footer
    className="container"
  >
    <div
      className="text"
    >
      (C) 2022 miyado
    </div>
    <a
      className="link"
      href="/contact"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      contact
    </a>
  </footer>
</div>
`;

exports[`Storyshots template/OneColumn Primary 1`] = `
<div
  className="container"
>
  <header
    className="header"
  >
    <h1
      className="title"
    >
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        miyado.dev
      </a>
    </h1>
  </header>
  <div
    className="main"
  >
    <main
      className="main"
    />
  </div>
  <footer
    className="container"
  >
    <div
      className="text"
    >
      (C) 2022 miyado
    </div>
    <a
      className="link"
      href="/contact"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      contact
    </a>
  </footer>
</div>
`;

exports[`Storyshots template/TwoColumn Primary 1`] = `
<div>
  Todo
</div>
`;
